src/main/java/com/practice/service/UserService.java

```diff
@@ -0,0 +1,42 @@
+package com.practice.service;
+
+import com.practice.User;
+import org.springframework.stereotype.Service;
+import org.springframework.ui.Model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+@Service
+public class UserService {
+    private static List<User> users = new ArrayList<>();
+
+    public List<User> getAllUsers() {
+        return users;
+    }
+
+    private static int todosCount = 0;
```

Comment:
Variable name `todosCount` doesn't make sense. We are using to store User ID sequence. Why used this name?